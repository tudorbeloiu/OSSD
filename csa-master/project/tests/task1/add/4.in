3
1
1
11
8170
1
1
21
20
1
1
31
12
