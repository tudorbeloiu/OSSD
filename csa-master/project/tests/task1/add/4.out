11: (0, 1021)
21: (0, 0)
31: (1022, 1023)
